assets/src/blocks/Gallery/GalleryEditor.js

```diff
@@ -124,6 +124,11 @@ const renderView = (attributes, setAttributes) => {
 
   const { images } = useGalleryImages({ multiple_image, gallery_block_focus_points }, layout);
 
+  const galleryProps = {
+    images: images || [],
+    onImageClick: () => {},
```

Comment:
This function receives an index (as a `number`) within the props.

Reply:
It doesn't really matter, it's just about not having it `null` or `undefined` so that it doesn't cause any errors